test/static/turtle_article.html

```diff
@@ -11,6 +11,8 @@
 <meta name="description" content="Exposition on the awesomeness of turtles"/>
 <meta name="keywords" content="turtles, are, awesome" />
 
+<link rel="apple-touch-icon" href="turtleapple.png" sizes="72x72">
+<link rel="icon" href="turtle.png" sizes="18x18" type="image/png">
```

Comment:
What happens if multiple icons are listed? Maybe should have test for this as well.